tools/scripts/warp.jlink.commands

```diff
@@ -1,6 +1,6 @@
 power on perm
 exec EnableRemarks
 unlock kinetis
-loadfile <full-path-to-warp-firmware>/build/ksdk1.1/work/demos/Warp/armgcc/Warp/release/Warp.srec
+loadfile /Users/hamid/warp-firmware-resolved-sleep-issue/Warp-firmware/build/ksdk1.1/work/demos/Warp/armgcc/Warp/release/Warp.srec
```

Comment:
Please remove this change from the PR. The change adds a variable for your local environment so it will break things for other people.